src/utils/utils.js

```diff
@@ -5,12 +5,12 @@ export const consolePrefix = 'SweetAlert2:'
  * @param arr
  */
 export const uniqueArray = (arr) => {
-  const result = []
-  for (var i in arr) {
-    if (result.indexOf(arr[i]) === -1) {
-      result.push(arr[i])
+  var result = []
+  arr.map(i => {
+    if (result.indexOf(i) === -1) {
```

Comment:
Hey, while you're fixing this function, could you make this line more readable? `if (!result.includes(i))`

Reply:
`Array.prototype.includes` isn't suported at all in IE 11: http://kangax.github.io/compat-table/es2016plus/#test-Array.prototype.includes

Reply:
@toverux Ah, yeah, I always forget that Babel does not (and can not) do anything about the new methods. I should have remembered issue #493.

@brian-kilpatrick Ignore my previous comment.

Reply:
It can, after all it has full access on the AST and can apply any transforms :) 
https://www.npmjs.com/package/babel-plugin-array-includes
Didn't thought about it immediately because I use TypeScript and it does not have plugins like that (yet).
Whether it's necessary to include or not a new dependency just for that  is another question :smile:

Reply:
I checked out that package. Check out the "Pitfalls" section. This kind of thing would work easily on a TypeScript AST, but with JS it is more difficult to determine if `result` is an array or something else. This is probably why this transform isn't included in the mainstream babel presets.

Reply:
Oh yeah, of course. I'm already too much used to static typing :sweat_smile: